Packs/CommonScripts/Scripts/SearchIncidentsV2/SearchIncidentsV2.py

```diff
@@ -79,7 +79,7 @@ def search_incidents(args: Dict):   # pragma: no cover
         args.pop('trimevents')
 
     # handle list of ids
-    if args.get('id'):
+    if args.get('id') and isinstance(args.get('id'), str):
```

Comment:
What if this script is a part of Playbook, and the input args['id'] is List[int]? 
I thinks in this case we will need to create comma separated list of ids as well. 
In addition, if args['id'] is int, does the automation `getIncidents` will work or id should be string?

Reply:
added a generic workaround for all cases (also UTs).